jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/impl/factory/DynAttrPropertyFilterComponentGenerationStrategy.java

```diff
@@ -139,7 +140,7 @@ protected Component createComponentInternal(ComponentGenerationContext context,
     protected AbstractField<?, ?> createUnaryField(@SuppressWarnings("unused") ComponentGenerationContext context) {
         //noinspection unchecked
         JmixComboBox<Boolean> component = uiComponents.create(JmixComboBox.class);
-        ComponentUtils.setItemsMap(component, ImmutableMap.of(
+        ComponentUtils.setItemsMap(component, Map.of(
```

Comment:
@KartnDev Why do we need to replace ImmutableMap with Map?

Reply:
Less code, more readable